packages/db/src/meta/interface.ts

```diff
@@ -34,15 +33,9 @@ export const forAttachAndSchema = <C extends Collections>(options: {
 }) => {
   const { attach, schema } = options;
 
-  const connect = (
-    config: TruffleConfig | ConnectOptions<C> | undefined
-  ): Db<C> => {
-    // TODO: get rid of @truffle/config stuff and rename field to "directory"
+  const connect = (config: ConnectOptions<C> | undefined): Db<C> => {
```

Comment:
Oh also, let's call this `options` since there's no longer anything named config here.

```suggestion
  const connect = (options: ConnectOptions<C> | undefined): Db<C> => {
```